datanode/entities/market.go

```diff
@@ -239,9 +241,18 @@ func liquidityMonitoringParametersFromProto(lmp *vega.LiquidityMonitoringParamet
 		}
 	}
 
+	tr := float64(0)
+	var err error
+	if len(lmp.TriggeringRatio) > 0 {
+		tr, err = strconv.ParseFloat(strings.TrimSpace(lmp.TriggeringRatio), 64)
+		if err != nil {
+			return LiquidityMonitoringParameters{}, err
+		}
+	}
+
 	return LiquidityMonitoringParameters{
 		TargetStakeParameters: tsp,
-		TriggeringRatio:       lmp.TriggeringRatio,
+		TriggeringRatio:       tr,
```

Comment:
Does that get save into postres as float? any chance it could get a different representation and fuck up the datanode snapshots accross different nodes?

Reply:
We may want to use decimal or string here too?

Reply:
Yes, I missed that, sorry.
Here I added the change to have the `TrigerringRatio` as a string inside the datanode object as well - https://github.com/vegaprotocol/vega/pull/7222/commits/2d223ec15e292897001a794aa99bd00d630636c4
So in that way we receive it as a string from the proto, do nothing on it - insert it as a string in the datanode object and in the DB
So we will have it as a decimal.Decimal object only in the core 
https://github.com/vegaprotocol/vega/pull/7222/commits/2d223ec15e292897001a794aa99bd00d630636c4